### PR TITLE
Add UUID suffix to Eua files

### PR DESCRIPTION
**Pull request by piotrekpasciak**

### CODE REVIEW

File: app/uploaders/eua_uploader.rb [hunk 1]
```diff
@@ -5,6 +5,10 @@ def store_dir
     "EUAs/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
   end
 
+  def filename
+    "#{original_filename.to_s.split('.').first}-#{SecureRandom.uuid}.#{file&.extension}"
+  end
```
Comment (alkeshrails) [inline app/uploaders/eua_uploader.rb:10]: where its call and get `original_filename` and `file` value?

Comment (skbly7) [reply app/uploaders/eua_uploader.rb:10]: @alkeshrails  It is coming from CarrierWave class. https://github.com/carrierwaveuploader/carrierwave/wiki/How-to:-Customize-your-version-file-names

@piotrekpasciak might be better to use `File.basename` or something here?

Comment (piotrekpasciak) [reply app/uploaders/eua_uploader.rb:10]: I'll check how `File.basename` works.

Comment (piotrekpasciak) [reply app/uploaders/eua_uploader.rb:10]: Where did you get `File.basename`? There is nothing like that.

Comment (skbly7) [reply app/uploaders/eua_uploader.rb:10]:
```
>> file = "/home/gumby/work/ruby_file.with.dot.rb" #=> "/home/gumby/work/ruby_file.with.dot.rb"
>> File.basename(file, File.extname(file)) #=> "ruby_file.with.dot"
```

Reference: https://stackoverflow.com/a/40846364/2529149

**Pull request merged**